includes/admin/payments/orders.php

```diff
@@ -905,6 +905,7 @@ function edd_order_details_attributes( $order ) {
 					<?php
 					echo EDD()->html->select( array(
 						'name'             => 'edd-payment-time-min',
+						'id'               => 'edd-payment-time-min',
```

Comment:
I wonder if we should change the current "Time" `label` to a `span` and then have a visually hidden `label` for "Hour" and "Minute" for these two selects. @robincornett?

Reply:
> In the "Customer" tab of "Order Details" if you click "Create New Customer", these need to be switched to `label`:
> 
> <img alt="Screen Shot 2020-04-16 at 4 08 32 PM" width="431" src="https://user-images.githubusercontent.com/491124/79505119-eb58a800-8001-11ea-9a43-c72faf529f21.png">
> 
> Let's add a visually hidden (`.screen-reader-text`) `label` to:
> 
> <img alt="Screen Shot 2020-04-16 at 4 46 15 PM" width="404" src="https://user-images.githubusercontent.com/491124/79505179-00353b80-8002-11ea-8c7b-6b067b2a1877.png">

I _think_ the directions for these are backwards.  I added `label`s to the "Customer" tab of "Order Details" -> "Create New Customer".  They lost their `strong` and some formatting.  Do you want me to deal with this now or after #7739?

![image](https://user-images.githubusercontent.com/18077352/79510168-b2b0d280-7ff1-11ea-8866-c100e6507e10.png)

Reply:
And I added `.screen-reader-text` to `edd-payment-time-hour` and `edd-payment-time-min`, but I can't figure out where the `style="display: none"` is coming from.

Reply:
@LisaCee The `style="display: none;"` on the actual `<select>` element is from the [Chosen](https://harvesthq.github.io/chosen/) script which renders an "enhanced" select box and hides the original. 

The hidden label changes look good, but I don't know if we should keep the `<label>` tag around "Time" now, since it duplicates the hidden "Hour" label now. I think it's safe to change that to `span`.

Reply:
@LisaCee the `display:none;` is being added by the `chosen` script to replace the standard dropdown.

Question for both of you (ping @spencerfinnell): how about we consider changing the hour and minute fields to `number` inputs, please? This seems like an ideal use for them because we can set a minimum and a maximum value and it's totally a value which can be changed incrementally.

Also, thinking through our discussions about radio inputs and grouped controls, I did some more digging and I think the hour/minute inputs would benefit from being "grouped" using a `<fieldset>` as they are interdependent.
- see especially the [SSN example](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA17) on the W3 site
- [address field](https://www.w3.org/WAI/WCAG21/Techniques/html/H71) as an example as well, so maybe we should consider doing the same with our address field groups
- consider using `aria-labelledby` to provide additional context for [fields in groups](https://www.w3.org/WAI/tutorials/forms/grouping/#associating-related-controls-with-wai-aria)

Reply:
@robincornett `number` does seem like a reasonable choice for these inputs. Can you open  a separate issue for that so we follow up with those adjustments at a later date?